packages/scaffolding-cli/templates/src/ssr/compositions/blog/index.test.cy.ts

```diff
@@ -0,0 +1,40 @@
+/*
+Every test you write will include selectors for elements. Write selectors that are resilient to changes.
+❌NEVER target elements based on CSS attributes such as: id, class, tag (e.g. cy.get('button').click(), cy.get('.btn.btn-large').click())
+❕SPARINGLY target elements on textContent if it won't change (e.g. cy.contains('Submit').click())
+✅ALWAYS add data-* attributes to make it easier to target elements
+*/
+/// <reference types="cypress" />
+
+/**
+ * @type {Cypress.PluginConfig}
+ **/
+
+describe("/Blog", () => {
+    beforeEach(() => {
+        cy.fixture("get-menu-response.json").as("menuResponse")
+        cy.server()
+        cy.route({
+            method: "GET", // Route all GET requests
+            url: "/menu*", // that have a URL that matches '/menu'
+            response: "@menuResponse", // and force the response to be: []
+        }).as("getStubbedMenu")
+        cy.visit("")
+    })
+
+    it("can navigate to /blog page when clicking on the blog button on header", () => {
+        cy.wait("@getStubbedMenu")
+        cy.get("[data-testid=blogs_button]").then($button => {
+            if ($button.is(":visible")) {
+                assert.isOk("blog button exists and active")
+            } else {
+                assert.isNotOk("blog button does not exist")
```

Comment:
Should only assert on your expectation. The error from a failed assertion will tell you if you it doesn't exist.